src/puppetlabs/puppetdb/config.clj

```diff
@@ -245,7 +245,10 @@
   "Checks that `vardir` is specified, exists, and is writeable, throwing
   appropriate exceptions if any condition is unmet."
   [config]
-  (let [vardir (io/file (get-in config [:global :vardir]))]
+  (let [vardir (some-> (get-in config [:global :vardir])
+                       io/file
```

Comment:
Is this `io/file` necessary? https://github.com/Raynes/fs/blob/master/src/me/raynes/fs.clj#L58